### ISSUE TITLE
 Variable $version of type String!

### ISSUE THREAD

**Issue by cricketsamya** [SmartsquareGmbH/delete-old-packages#19]
```
Run smartsquaregmbh/delete-old-packages@v0.3.1
Warning: Dry run is set. No package versions will be actually deleted.
Fetching packages
Error: Variable $version of type String! was provided invalid value
```
version value I tried were "3.2-SNAPSHOT", 3.2-SNAPSHOT, '3.2-SNAPSHOT' nothing works.

**Comment by rubengees** [comment]
Can you post (at least part of) your configuration?

**Comment by cricketsamya** [comment]
```
   uses: smartsquaregmbh/delete-old-packages@v0.3.1
        with:          
          dry-run: true        
          version: 3.2-SNAPSHOT
          names: |
```

```
   uses: smartsquaregmbh/delete-old-packages@v0.3.1
        with:          
          dry-run: true        
          version: "3.2-SNAPSHOT"
          names: |
```

```
   uses: smartsquaregmbh/delete-old-packages@v0.3.1
        with:          
          dry-run: true        
          version: '3.2-SNAPSHOT'
          names: |
```

```
uses: smartsquaregmbh/delete-old-packages@v0.3.1
        with:
          dry-run: true        
          version: ${{ github.event.inputs.delete_version }}
          names: |

```

These are all possible values gives me same error.

**Comment by fabian-kotschenreuther** [comment]
Same problem for me.

**Comment by rubengees** [comment]
Should work again with version `0.3.2`, please let me know if it does not :slightly_smiling_face:.

**Comment by Niek** [comment]
A very similar error since this morning:
```
Request failed due to following response errors: - Type mismatch on variable $packageVersionId and argument packageVersionId (String! / ID!)
```

Any workaround?

**Comment by maneetgoyal** [comment]
Getting a similar error:

```
Error: Type mismatch on variable $packageVersionId and argument packageVersionId (String! / ID!)
```

Using v0.3.1. @Niek Were you able to find a solution?

**Comment by maneetgoyal** [comment]
Ref: https://github.com/actions/delete-package-versions/issues/66

**Comment by nickdandakis** [comment]
Confirming the same error on our side as well. @rubengees created PR #22 to resolve this

**Comment by rubengees** [comment]
Released as v0.4.0. Thanks @nickdandakis!
